### PR TITLE
Providing view of user board

### PR DESCRIPTION
**Pull request by igasm**

### CODE REVIEW

File: client/src/main/java/battleships/RootLayoutController.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    private void addPlayerBoardView(){
+        try {
+            FXMLLoader loader = new FXMLLoader();
+            loader.setLocation(App.class.getResource(PLAYER_BOARD_VIEW_FXML));
+
+            BorderPane initLayout = loader.load();
+            borderPane.setLeft(initLayout);
+
+        } catch (IOException e) {
+            e.printStackTrace();
```
Comment (M-Aarsman) [inline client/src/main/java/battleships/RootLayoutController.java]: in future logger

File: client/src/main/java/battleships/game/Board.java [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+class Board {
+
+    private static final int BOARD_SIZE = 100;
+    private static final int RECTANGLE_SIZE = 30;
+    private static final String MARK_FOR_SHOT = "x";
+
+    List<Field> fields;
+
+    private Map<FieldState, Color> colorMap;
```
Comment (M-Aarsman) [inline client/src/main/java/battleships/game/Board.java]: can be final

File: client/src/main/java/battleships/game/Board.java [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+        map.put(FieldState.BROKEN_SHIP_PART, Color.YELLOW);
+        map.put(FieldState.SUNK_SHIP_PART, Color.RED);
+        List<Field> emptyFields = IntStream.range(0, BOARD_SIZE).mapToObj(p -> new Field(p)).collect(Collectors.toList());
+        return new Board(emptyFields, map);
+    }
+
+    /**
+     * created for temporary use
+     */
+    void generateExample(){
```
Comment (M-Aarsman) [inline client/src/main/java/battleships/game/Board.java]: this function can be divided to at least 4 private

File: client/src/main/java/battleships/logging/LoggingController.java [hunk 4]
```diff
@@ -97,5 +100,18 @@ private void initialize(LanguageLoadOption languageLoadOption) {
...
         assignKeyTranslation();
     }
+
+    public void setMainApp(App mainApp) {
+        this.mainApp = mainApp;
+    }
+
+    private void invalidLoggingDataAlert(){
+        Alert alert = new Alert(Alert.AlertType.WARNING);
+        alert.setTitle("Warning Dialog");
```
Comment (M-Aarsman) [inline client/src/main/java/battleships/logging/LoggingController.java]: magic string -> Krzysztof

File: client/src/main/java/battleships/logging/LoggingController.java [hunk 5]
```diff
@@ -97,5 +100,18 @@ private void initialize(LanguageLoadOption languageLoadOption) {
...
+        alert.setHeaderText("Logging data is invalid");
```
Comment (M-Aarsman) [inline client/src/main/java/battleships/logging/LoggingController.java]: magic string -> Krzysztof

File: client/src/main/java/battleships/logging/LoggingController.java [hunk 6]
```diff
@@ -97,5 +100,18 @@ private void initialize(LanguageLoadOption languageLoadOption) {
...
+        alert.setContentText("...");
```
Comment (M-Aarsman) [inline client/src/main/java/battleships/logging/LoggingController.java]: magic string -> Krzysztof

File: client/src/test/java/battleships/game/FieldTest.java [hunk 7]
```diff
@@ -0,0 +1,49 @@
+package battleships.game;
+
+import org.testng.annotations.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class FieldTest {
+
+    @Test
+    public void givenPosition_whenCreatingField_itsPositionShouldBeEqualAsGiven(){
```
Comment (M-Aarsman) [inline client/src/test/java/battleships/game/FieldTest.java]: parametrize maybe

**Comment by igasm** [comment]
I've changed variables to final, as you requested or just removed it. I split function generating ship into smaller ones. I've parametrized tests in FieldTest class. But I didn't extract strings that you mentioned to constants, because this are just messages.

**Pull request merged**